<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loopover</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/icons/ico/icon.ico</normaloff>:/icons/ico/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonScrumble">
        <property name="text">
         <string>Scrumble</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTableView" name="tableView">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
